mailpoet/lib/Subscribers/InactiveSubscribersController.php

```diff
@@ -79,8 +79,6 @@ private function deactivateSubscribers(Carbon $thresholdDate, int $batchSize, ?i
       return false;
     }
 
-    // We take into account only emails which have at least one opening tracked
-    // to ensure that tracking was enabled for the particular email
```

Comment:
Yeah, this comment was confusing me in the previous commit. I'm also thinking the `inactive_task_ids` as a table name here is rather confusing as well. From what it seems this does is fetching all task IDs "from threshold date up to yesterday", which are only candidates in the inactivity period.

No need to change though, as this was not added now.